types/types.go

```diff
@@ -397,6 +399,15 @@ func GetRecurringJobLabelValueMap(labelType, recurringJobName string) map[string
 		GetRecurringJobLabelKey(labelType, recurringJobName): LonghornLabelValueEnabled,
 	}
 }
+
+func GetOrphanLabels(node, diskUUID, orphanName string) map[string]string {
```

Comment:
1. Have you used `orphanName`?
2. This CRD is for general cases. But the labels here are for replica directories only. In other words, you need to label the type first.
3. Add the related label when the input is not empty.